content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -74,13 +77,89 @@ If you notice data is missing in your bucket, do the following:
 - Verify the timestamps match the [precision parameter](/influxdb/cloud/write-data/#timestamp-precision).
 - Minimize payload size and network errors by [optimizing writes](/influxdb/cloud/write-data/best-practices/optimize-writes/)
 
-### Troubleshoot rejected points
+### Review rejected points
+
+InfluxDB may have rejected points even if the HTTP request returned "Success".
+Your organization's `_monitoring` bucket logs rejected data points and any parsing errors associated with them.
+To get a log of rejected points, query for the [`rejected_points` measurement](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) and the [`count` field](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
+Each entry has a `count` equal to `1`.
+
+```js
+from(bucket: "_monitoring")
+  |> range(start: -1h)
+  |> filter(fn: (r) => r._measurement == "rejected_points")
+  |> filter(fn: (r) => r["_field"] == "count")
+```
+
+InfluxDB returns `rejected_points` entries that contain the `bucket` where the rejection occurred and the `reason` for the rejection.
+If a parsing error occurred, the entry will contain an `error` field with details.
+
+```text
+rejected_points,bucket=YOUR_BUCKET,field=YOUR_FIELD,gotType=Float, \
+  measurement=YOUR_MEASUREMENT,reason=type\ conflict\ in\ batch\ write, \
+  wantType=Integer count=1i 1627906197091972750
+```
+
+If the data you want to write contains multiple fields in a data point, InfluxDB creates a `rejected_points` entry for each [series](/influxdb/cloud/reference/key-concepts/data-elements/#series) that failed.
+For example, the following line protocol data has an `airSensors` measurement and three fields (`temperature`, `humidity`, and `co`).
+
+```
+airSensors,sensorId=TLM0201 temperature=73.97,humidity=35.23,co=0.48 1637014074
+
+```
+
+If you try to write this data to a bucket that has the [`explicit` type](/influxdb/cloud/organizations/buckets/bucket-schema/) and doesn't have an `airSensors` schema, the `/api/v2/write` InfluxDB API returns an error and the following data:
```

Comment:
```suggestion
If you try to write this data to a bucket with an [explicit schema](/influxdb/cloud/organizations/buckets/bucket-schema/) that doesn't include the `airSensors` measurement, the `/api/v2/write` InfluxDB API returns an error and the following data:
```

Reply:
@sanderson, that's not exactly accurate. "bucket schemas" are actually *measurement schemas*.